File: src/main/java/com/sce/platform/auth/dto/LoginResponse.java
package com.sce.platform.auth.dto;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.util.List;
import java.util.UUID;

@Getter
@Setter
@NoArgsConstructor
public class LoginResponse {
    private UUID usuarioId;
    private String nombreUsuario;
    private String nombres;

    private TenantDisponibleResponse tenantSeleccionado;
    private List<TenantDisponibleResponse> tenants;
}

File: frontend/src/app/auth/models/login-response.ts
import { TenantDisponibleResponse } from './tenant-disponible-response';


export interface LoginResponse {
  usuarioId: string;
  nombreUsuario: string;
  nombres: string;
  tenantSeleccionado?: TenantDisponibleResponse;
  tenants?: TenantDisponibleResponse[];
  token?: string;
  selectionToken?: string;
}
